templates/style.php

```diff
@@ -414,3 +414,31 @@
 	position: absolute;
 	<?php echo is_rtl() ? 'left' : 'right'; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?>: 16px;
 }
+
+/* Dark Theme */
+
+:root {
+	color-scheme: light dark;
+}
+
+@media (prefers-color-scheme: dark) {
+	body,
+	.amp-wp-article {
+		background: #202020;
+		color: #fff;
+	}
+
+	.amp-wp-tax-category,
+	.amp-wp-tax-tag,
+	.amp-wp-meta,
+	span.amp-site-title,
+	.amp-wp-title {
+		color: #fff;
+	}
+
+	a,
+	a:visited,
+	.amp-wp-comments-link a {
+		color: #93B3DB;
+	}
+}
```

Comment:
The build is failing because the file doesn't end in a newline: https://travis-ci.org/ampproject/amp-wp/jobs/587871984#L782-L783

Reply:
Pushed another commit to add the newline